app/javascript/bundles/UI/components/ProgressBar/ProgressBar.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * ProgessBar.jsx - Created by David Redó
+ * 
+ * First edit: 19th Sep. 2018 - David Redó
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import css from './ProgressBar.scss';
+import classnames from 'classnames';
+
+class ProgressBar extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {
```

Comment:
We should use the simple props pass through, like in [the Alert Component] (https://github.com/infusionvlc/ConexionFelina/blob/react/app/javascript/bundles/UI/components/Alert/Alert.jsx#L8) so we don't need to edit the state with a simple component, at least not yet.